configs/caas/docker-compose.yaml.ctmpl

```diff
@@ -36,25 +44,43 @@ services:
     restart: always
   cromwell:
     image: broadinstitute/cromwell:{{$tag}}
+    logging:
+      driver: syslog
+      options:
+        tag: "cromwell-app"
     environment:
-      JAVA_OPTS: -Dscala.concurrent.context.minThreads=50 -Dscala.concurrent.context.numThreads=50 -Dscala.concurrent.context.maxThreads=50 -Xmx20g -Xms20g -Djavax.net.ssl.keyStore=/etc/ssl/cromwell.keystore -Djavax.net.ssl.keyStorePassword=changeit -Djavax.net.ssl.trustStore=/etc/ssl/cromwell.truststore -Djavax.net.ssl.trustStorePassword=changeit -Dconfig.file=/etc/cromwell.conf
+      JAVA_OPTS: -Dscala.concurrent.context.minThreads=50 -Dscala.concurrent.context.numThreads=50 -Dscala.concurrent.context.maxThreads=50 -Xmx20g -Xms20g -Dconfig.file=/etc/cromwell.conf
       CROMWELL_ARGS: server
     volumes:
       - /etc/localtime:/etc/localtime:ro
       - {{$dir}}/cromwell.conf:/etc/cromwell.conf:ro
       - {{$dir}}/cromwell-account.json:/etc/cromwell-account.json:ro
-      - {{$dir}}/mysql01.jks:/etc/ssl/cromwell.keystore:ro
-      - {{$dir}}/mysql_trust.jks:/etc/ssl/cromwell.truststore:ro
     restart: always
     expose:
       - "8000"
       - "8001"
     links:
       - logger
+      - sqlproxy:sqlproxy
     volumes_from:
       - logger
+  sqlproxy:
+    image: broadinstitute/cloudsqlproxy:1.11_2018117
+    logging:
+      driver: syslog
+      options:
+        tag: "cromwell-sqlproxy"
+    env_file:
+      - {{$dir}}/sqlproxy.env
+    volumes:
+      - {{$dir}}/cromwell-account.json:/etc/sqlproxy-service-account.json
+    restart: always
   logger:
     image: broadinstitute/fluentd-gcp:latest
+    logging:
+      driver: syslog
+      options:
+        tag: "cromwell-logger"
```

Comment:
I actually don't know what it is doing since this config was constructed by someone else - there are several configurations that I don't quite understand.  I did not want to remove it since I did want to take on the additional work of figuring out what configs are needed and what were not.